roles/online_docs/templates/mkdocs/docs/logins-windows.bat

```diff
@@ -0,0 +1,101 @@
+#jinja2: trim_blocks:False
```

Comment:
This is not a MarkDown document that should be converted into HTML. Please relocate to for example`roles/online_docs/templates/attachments/`